File: app/src/main/java/com/mdgd/pokemon/models/repo/schemas/OfficialArtwork.java
package com.mdgd.pokemon.models.repo.schemas;

import androidx.room.ColumnInfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class OfficialArtwork {

    @ColumnInfo(name = "OfficialArtworkFrontDefault")
    @Expose
    @SerializedName("front_default")
    private String frontDefault;

    public String getFrontDefault() {
        return frontDefault;
    }

    public void setFrontDefault(String frontDefault) {
        this.frontDefault = frontDefault;
    }

}

File: models_impl/src/main/java/com/mdgd/pokemon/models_impl/repo/schemas/OfficialArtwork.kt
package com.mdgd.pokemon.models_impl.repo.schemas

import androidx.room.ColumnInfo
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class OfficialArtwork {
    @ColumnInfo(name = "OfficialArtworkFrontDefault")
    @Expose
    @SerializedName("front_default")
    var frontDefault: String? = null
}
